io-backend/src/main/java/agh/io/iobackend/model/GameRoomSequenceIdGenerator.java

```diff
@@ -0,0 +1,21 @@
+package agh.io.iobackend.model;
+
+import org.hibernate.HibernateException;
+import org.hibernate.engine.spi.SharedSessionContractImplementor;
+import org.hibernate.id.enhanced.SequenceStyleGenerator;
+
+import java.io.Serializable;
+
+public class GameRoomSequenceIdGenerator extends SequenceStyleGenerator {
+    @Override
+    public Serializable generate(
+            SharedSessionContractImplementor session, Object object
+    ) throws HibernateException {
+        Long seed = (Long) super.generate(session, object);
```

Comment:
Czy to nam jakoś zapewnia że te ID zawsze będą unikatowe? Rozumiem że prawdopodobieństwo że wylosujemy dwa takie same jest praktycznie równe 0?

Reply:
Tak, dokładnie. No z tego co pamiętam z asd, to zaczną się powtarzać dopiero jak przekroczymy to modulo, czyli po 2^32 pokojach, co jest raczej nieosiągalne nawet jeśli nasza gra okaże się hitem sezonu

Reply:
okej, to jak dla mnie jest gites, jak tylko rozwiążesz konflikty z masterem to można mergować